h/assets.py

```diff
@@ -1,151 +1,19 @@
-import configparser
-import json
-import os
+import os.path
```

Comment:
```suggestion
"""Add a view for getting static assets with cache busting."""

import os.path
```

Reply:
Done, though with a slight revision to the wording.